app/src/main/java/org/fossasia/openevent/general/social/SocialLinksFragment.kt

```diff
@@ -9,14 +9,8 @@ import androidx.core.view.isVisible
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.recyclerview.widget.LinearLayoutManager
-import com.google.android.material.snackbar.Snackbar
-import kotlinx.android.synthetic.main.fragment_social_links.eventHostDetails
-import kotlinx.android.synthetic.main.fragment_social_links.socialLinksRecycler
-import kotlinx.android.synthetic.main.fragment_social_links.socialLinksCoordinatorLayout
-import kotlinx.android.synthetic.main.fragment_social_links.view.progressBarSocial
-import kotlinx.android.synthetic.main.fragment_social_links.view.socialLinksRecycler
-import kotlinx.android.synthetic.main.fragment_social_links.view.socialLinkReload
-import kotlinx.android.synthetic.main.fragment_social_links.view.socialNoInternet
+import kotlinx.android.synthetic.main.fragment_social_links.*
+import kotlinx.android.synthetic.main.fragment_social_links.view.*
```

Comment:
Wildcard imports

Reply:
It passed spotless check so thought its fine. Doing it

Reply:
resolved